themes/bootstrap3/templates/search/bulk-action-buttons.phtml

```diff
@@ -1,26 +1,26 @@
 <?php if (isset($this->showCheckboxes) && $this->showCheckboxes): ?>
-  <div class="bulkActionButtons hidden-print">
+  <nav class="bulkActionButtons">
```

Comment:
Is hidden-print being removed intentionally?

Reply:
Yes, I meant to tie it into the LESS but missed this step. Thank you!

Reply:
Nevermind, I already added it here: https://github.com/crhallberg/vufind/blob/bulk-normalize/themes/bootstrap3/less/components/search.less#L57

Reply:
Aha, thanks for clarifying that. At this point, what are the circumstances under which we would use .hidden-print in the template vs. a targeted `display:none` in the LESS? Just wondering if we have any inconsistencies that should be normalized....